spec/factories/miq_alert_status_label.rb

```diff
@@ -0,0 +1,11 @@
+FactoryGirl.define do
+  factory :miq_alert_status_label do
```

Comment:
IMHO a factory for a non-AR class is confusing.  I'd prefer to see normal `MiqAlertStatusLabel.new` in tests. (can add `initialize` or define it as a `Struct` to make that easier.)

Reply:
The factory is needed because there is a [test](https://github.com/ManageIQ/manageiq/blob/master/spec/models/factory_girl_spec.rb) that checks that there are factories for all the classes in the `app/models` directory.